### ISSUE TITLE
Feature Request: add ability for a TEXTURE block to modify shader properties

### ISSUE THREAD

**Issue by ChrisAdderley** [blowfishpro/B9PartSwitch#149]
It would be great to be able to modify shader properties within a TEXTURE or a SUBTYPE block, in the same way that the Restock things you wrote do, eg. 
```
COLOR_PROPERTY
    {
      name = _SpecColor
      color = 0.753, 0.753, 0.753, 1.0
    }

    FLOAT_PROPERTY
    {
      name = _Shininess
      value = 0.3
    }
```
This would increase the flexibility of texture switching significantly.

**Comment by blowfishpro** [comment]
Will probably implement this as an alternative to the `TEXTURE` blocks, that was done before I really understood the scope of what needed to be changed

**Comment by ChrisAdderley** [comment]
Sounds good to me!

**Comment by blowfishpro** [comment]
Implemented in #185
